benefits/static/css/styles.css

```diff
@@ -263,13 +266,6 @@ footer {
     margin-top: 8rem;
     margin-bottom: 7rem;
   }
-
-  .container.content input[type="submit"],
-  .container.content .btn {
-    width: 70%;
-    margin-left: 15%;
-    margin-right: 15%;
-  }
```

Comment:
Are you sure these are ok to remove? i.e. is it possible there were other pages that should have these styles?